aries-backchannels/acapy/acapy_backchannel.py

```diff
@@ -301,6 +301,9 @@ def get_agent_args(self):
         result.append(("--plugin", "universal_resolver"))
         result.append(("--plugin-config", "/data-mount/plugin-config.yml"))
 
+        #HARD CODE REDIS RN
```

Comment:
Does this impact the acapy and acapy-main backchannels?  E.g., is the hard-coded value in this shared script OK?

Reply:
I think acapy just sees the parameter, but not the plugin, and prints a warning\, but I think I can leave it as is (see the universal_resolver above).